docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -1,6 +1,39 @@
 import { Alert, Tabs, TabItem } from '@aws-amplify/ui-react';
 import { TerminalCommand } from '@/components/InstallScripts';
 
+## 3.x to 4.x
+### Installation
+Install the 4.x version of the `@aws-amplify/ui-angular` library.
+
+<Tabs>
+<TabItem title="npm">
+<TerminalCommand command="npm install @aws-amplify/ui-angular@4.x" />
+</TabItem>
+<TabItem title="yarn">
+<TerminalCommand command="yarn add @aws-amplify/ui-angular@4.x" />
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+`@aws-amplify/ui-angular@4.x` introduces the following breaking changes:
+
+#### 1. `@aws-amplify/ui-angular@4.x` bumps minimum Angular version to 14
+
+If your application is on Angular 13 or older, please migrate to Angular 14+ as per [official guide](https://update.angular.io).
+
+*Note*: Going forward, `@aws-amplify/ui-angular` will have the same version support as Angular's official LTS [support window](https://angular.io/guide/releases#actively-supported-versions).
```

Comment:
Nit: maybe to make it abundantly clear he we can say something to the effect of `starting from version x` isntead of `going foward`?

Reply:
I think we can remove this note as we have aligned to support Angular's official LTS with a note in previous version upgrade.

Reply:
Or we could just say that we support the current LTS versions in our Angular installation page?

Reply:
I agree with @reesscot's suggestion.

Reply:
Updated in https://github.com/aws-amplify/amplify-ui/pull/4007/commits/cda137e4bd7d2c6cd8e8c0e360b59af3e354b205